caiman/source_extraction/volpy/mrcnn/model.py

```diff
@@ -2378,7 +2368,7 @@ def train(self, train_dataset, val_dataset, learning_rate, epochs, layers,
             validation_steps=self.config.VALIDATION_STEPS,
             max_queue_size=100,
             workers=workers,
-            use_multiprocessing=True,
+            use_multiprocessing=workers > 1,
```

Comment:
I see above you mention that Keras fails on Windows when using multiprocessing; is this still true of the keras layer in tensorflow? If not, the comment above (line 2363) should be removed. If not, is this code nonusable on windows with this change?

Reply:
I actually only tested the Mask R-CNN in VolPy on the linux system.